flight/Modules/Autotune/autotune.c

```diff
@@ -123,6 +123,8 @@ static void UpdateSystemIdent(const float *X, const float *noise,
 		relay.Noise[SYSTEMIDENT_NOISE_YAW]   = noise[2];
 	}
 	relay.Period = dT_s * 1000.0f;
+
+	relay.NumAfPredicts = predicts;
```

Comment:
Shouldn't `Af` be `Of`, or what does `Af` stand for? Also, would prefer it to be more human readable, e.g. `NumberOfPredictions`.